apps/site/src/views/Home/sections/FAQ/FAQ.tsx

```diff
@@ -1,11 +1,63 @@
+import { useEffect, useState } from "react";
 import Container from "react-bootstrap/Container";
+import Accordion from "react-bootstrap/Accordion";
+import styles from "./FAQ.module.scss";
+
+const santiy_url: string =
```

Comment:
Typo: should be "sanity" instead of "santiy".

Nitpick [repeating]: we tend to use camel case instead of snake case in JS/TS.

Reply:
fixed!